bricks/aem-logo-wall/aem-logo-wall.html

```diff
@@ -0,0 +1,97 @@
+<template>
+  <style>
+    @import url('/styles/bricks-common.css');
+
+    :host {
+      --spacing-s: 24px;
+      --spacing-l: 48px;
+      --spacing-xl: 56px;
+      --spacing-xxl: 80px;
+
+      position: relative;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      padding: var(--spacing-xl) 0;
+    }
+
+    h1, h2, h3, h4, h5, h6 {
```

Comment:
IMHO the values for such "general typography" styles should be defined outside of the component's code, maybe in a "global" set of CSS custom property definitions, which set the "theme" of the website.
Not a blocker for merging, we can discuss later.

Reply:
In the original site they were defined under ".logo-wall-title", which was applied to all h1..6 within the block, but as far as I understand it shouldn't be applied as a theme.

Reply:
Ah ok, if they are specific to a given block having them here (until we eventually remove Shadow DOM if that makes sense) is fine

Reply:
if we remove shadow dom, then we should make them more concrete by prefixing them with "aem-logo-wall", but it should still be only be applied to this brick.